package.json

```diff
@@ -41,7 +43,8 @@
     "tippy.js": "^6.3.7",
     "ts-pattern": "^4.0.5",
     "use-debounce": "^8.0.4",
-    "yup": "^0.32.11"
+    "yup": "^0.32.11",
+    "zod": "^3.23.8"
```

Comment:
Are these not doing the same thing. Validation?

Reply:
Yes, you're absolutely right; they work the same. The HubSpot App was written when we were still using `Formik` and `yup`. But now, in Deskpro, we're moving to `react-hook-form` and `zod` whenever possible. So, for these stories, I'll use `react-hook-form` and `zod` _(these are the two largest forms in this app)_. And as part of closing the tech debt, I'll rewrite the other three smaller forms.